src/System.Management.Automation/FormatAndOutput/common/PSStyle.cs

```diff
@@ -516,6 +684,17 @@ private PSStyle()
             FileInfo = new FileInfoFormatting();
         }
 
+        private static string ValidateNoContent(string text)
```

Comment:
"Validate" assumes a return true/false. Perhaps NormalizeContent is better name. The same for ValidateExtension.

Reply:
Normalize doesn't indicate it'll throw if the data isn't right.

Reply:
It is not strong rule. If it is important we could name the method as NormalizeContentOrThrow.

Reply:
I think previously where I added the period if it was missing, then `Normalize` would make sense but since it's only checking if the period is there, it seems to me validate is more accurate than normalize.  Perhaps we need another opinion here, @rjmholt ?

Reply:
`Validate` sounds more like an assertion that throws than a boolean to me &mdash; I think it fits here.

If it returned a boolean, I'd expect a predicate phrase like `IsContentFree()` or `ContainsNoContent()`.